src/frontend/src/lib/components/networks/NetworksTestnetsToggle.svelte

```diff
@@ -1,13 +1,27 @@
 <script lang="ts">
 	import { Toggle } from '@dfinity/gix-components';
+	import { goto } from '$app/navigation';
+	import { NETWORK_PARAM } from '$lib/constants/routes.constants';
 	import { testnetsEnabled } from '$lib/derived/settings.derived';
 	import { i18n } from '$lib/stores/i18n.store';
 	import { testnetsStore } from '$lib/stores/settings.store';
 
+	// TODO: create tests for this component once we have testId from GIX-CMP
+	// PR: https://github.com/dfinity/gix-components/pull/531
+
 	let checked: boolean;
 	$: checked = $testnetsEnabled;
 
-	const toggleTestnets = () => testnetsStore.set({ key: 'testnets', value: { enabled: !checked } });
+	const toggleTestnets = async () => {
+		// Reset network param, since the network is selectable only when testnets are enabled
+		if (checked) {
```

Comment:
I would maybe do this after the store is updated, not sure what's the most important?

Reply:
sure!